support/helper.py

```diff
@@ -123,15 +142,15 @@ def filter_dict(lst, key, value):
     return [d for d in lst if d[key].lower() == value]
 
 
-def dict_reverse_cumsum(d, col, sum_col):
-    df = pd.DataFrame(d)
+def dict_reverse_cumsum(lst, col, sum_col):
+    df = pd.DataFrame(lst)
     cols = df.columns
     data = pd.DataFrame(columns=cols)
     accounts = df['account_id'].unique().tolist()
     for account_id in accounts:
         temp = df[df['account_id'] == account_id]
         row = pd.DataFrame(temp[-1:].values, columns=cols)
-        row.at[0, col] = row.at[0, sum_col]*-1
+        row.at[0, col] = row.at[0, sum_col]
```

Comment:
Please, let's document our **sign convention** in our Git README.md. 

- negative sign = inflow = income --> money flowing into a bank account (e.g., payroll)
- positive sign = outflow = expenses --> money flowing out of bank account (e.g., purchase)

Or vice versa.

Reply:
negative means money leaving the account (outflow / expenses), while positive means inflow